### ISSUE TITLE
"Delete account" not accessible through keyboard navigation

### ISSUE THREAD

**Issue by connorlanigan** [bitwarden/clients#3401]
### Steps To Reproduce

1. Go to the settings
2. Using the <kbd>Tab</kbd> key, navigate to the item immediately before _"Delete account"_ (currently _"Unlock with PIN"_)
3. Press the <kbd>Tab</kbd> key once more

### Expected Result

The _"Delete account"_ link is focused.

### Actual Result

The focus skips the _"Delete account"_ link, and the _"PREFERENCES"_ section header is focused instead. The account can thus not be deleted without using a mouse.

### Screenshots or Videos

![screen recording of tabbing through the settings. the focus skips the "Delete account" item.](https://user-images.githubusercontent.com/9086585/187069535-faef1365-9623-4133-8452-860bd9927e5f.gif)

### Additional Context

The "Delete account" text is currently an `<a>` element (without an `href` attribute). Since it does not act as a link, but rather opens a new modal, it could potentially be implemented as a `<button>` instead. This way it would automatically be focusable.

### Operating System

Windows

### Operating System Version

11

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Version 2022.8.1

**Comment by patrickhlauke** [comment]
nice catch. i hadn't even clocked that that was a control in its current form. filed https://github.com/bitwarden/clients/pull/3406
